_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction 
+In [EO](https://github.com/objectionary/eo) a caching is used to speed up program execution.
```

Comment:
@Yanich96 
```txt
In [EO](https://github.com/objectionary/eo), caching is used to speed up program execution.
```

- "caching" is uncountable here, you don't need an article "a"
- You need a comma after "In EO"

Reply:
@Yanich96 "program execution" or "program compilation" ?